### PR TITLE
[Affiliate] Notification banner component

### LINKED ISSUE #943: [Affiliate] Add notification banner

### PR DESCRIPTION
**Pull request by matextrem**
# Summary
Add notification banner component in order to show different messages related to the Affiliate Status (#944)
Closes #943 

Screen Shot 2021-07-22 at 12 25 59

Screen Shot 2021-07-22 at 12 26 37

# Testing steps
Since it is not a integration and given the fact we don't have Storybook to list components, there is no visual feedback until it is integrated (#944)

### CODE REVIEW

File: src/custom/components/NotificationBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { Colors } from 'theme/styled'
+import { X } from 'react-feather'
+
+export interface BannerProps {
+  children: React.ReactNode | string
+  level: 'info' | 'warning' | 'error'
+  visible: boolean
+}
+
+const Banner = styled.div<{ isActive: any; level: 'info' | 'warning' | 'error' }>`
```
Comment (alfetopito) [inline src/custom/components/NotificationBanner/index.tsx:12]: `isActive` is a boolean, no?

```suggestion
const Banner = styled.div<{ isActive: boolean; level: 'info' | 'warning' | 'error' }>`
```

Comment (alfetopito) [reply src/custom/components/NotificationBanner/index.tsx:12]: Also, you could reuse the `level`.

Define a type `level` and  use it on BannerProps and here.

Alternatively, you can use `BannerProps` here:

```suggestion
const Banner = styled.div<Pick<BannerProps, 'visible' | 'level'>>`
```

Of course, you'll need to replace `isActive` with `visible` which is the same already anyway.

Also also :) I suggest renaming `visible` to `isVisible`

**Review by alfetopito — COMMENTED** [review]
I understand this is nowhere in the page to be tested, but I assume you at least tried it out in the page to see how it looks like and come up with the CSS?

In that case, could you add a screenshot for reference?

As for the colors themselves, I'll leave up to @biocom

**Review by alfetopito — APPROVED** [review]
Looking good :+1:

**Review by ramirotw — APPROVED** [review]

**Pull request merged**